app/models/vm_or_template.rb

```diff
@@ -496,12 +548,7 @@ def self.invoke_tasks_remote(options)
 
         $log.error("An error occurred while invoking remote tasks...Requeueing for 1 minute from now.")
         $log.log_backtrace(err)
-        MiqQueue.put(
-          :class_name  => base_class.name,
-          :method_name => 'invoke_tasks_remote',
-          :args        => [remote_options],
-          :deliver_on  => Time.now.utc + 1.minute
-        )
+        invoke_tasks_remote_queue(remote_options, Time.now.utc + 1.minute)
```

Comment:
Yeah .... That's how the SOAP version was written too. We could add something to the options to indicate a retry count, but I'm not sure how much we need to enhance this here.